test/ex_member_page_test.rb

```diff
@@ -2,19 +2,63 @@
 require_relative './test_helper'
 require_relative '../lib/ex_member_page.rb'
 
-describe MemberPage do
-  around { |test| VCR.use_cassette('GeorgeVella', &test) }
+describe ExMemberPage do
+  around { |test| VCR.use_cassette(url.split('/').last.split('-').map(&:capitalize).join(''), &test) }
```

Comment:
That's quite complex logic just to get a cassette name. Isn't there something simpler you could use?